bundles/org.openhab.binding.elerotransmitterstick/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -8,4 +8,25 @@
 	<description>This is the binding for Elero Transmitter Sticks.</description>
 	<connection>local</connection>
 
+	<discovery-methods>
+		<discovery-method>
+			<service-type>usb</service-type>
+			<match-properties>
+				<match-property>
+					<name>manufacturer</name>
+					<regex>(?i)elero gmbh.*</regex>
+				</match-property>
+			</match-properties>
+		</discovery-method>
+		<discovery-method>
+			<service-type>usb</service-type>
+			<match-properties>
+				<match-property>
+					<name>chipId</name>
+					<regex>0403:6001</regex>
```

Comment:
Wouldn't that trigger false positives similar to https://github.com/openhab/openhab-addons/issues/4421?

Reply:
Oops. Thanks for checking.